tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py36, py37, py38, py39
+envlist = py36, py37, py38, py39

```

Comment:
Yes, I agree 2.7 should be dropped (there is a bit of clean-up to do to fully remove six and python 2 specific code). I'll drop 3.6 too, and add 3.10 (and soon 3.11)